src/common/options.cc

```diff
@@ -6495,6 +6495,14 @@ std::vector<Option> get_rgw_options() {
         "the metadata sync parallelism as a shard can only be processed by a single "
         "RGW at a time"),
 
+    Option("rgw_curl_buffersize", Option::TYPE_INT, Option::LEVEL_DEV)
+    .set_default(16_K)
```

Comment:
is 16K the builtin default?

Reply:
As it's said in [CURLOPT_BUFFERSIZE explained](https://curl.se/libcurl/c/CURLOPT_BUFFERSIZE.html):
```text
This buffer size is by default CURL_MAX_WRITE_SIZE (16kB).
```